.vscode/settings.json

```diff
@@ -4,5 +4,7 @@
         "MD025": {
             "front_matter_title": ""
         }
-    }
-}
\ No newline at end of file
+    },
+    "files.associations": {
+        "*.workbook": "json"
+    },}
```

Comment:
is this valid json?  the preview/diff tool has a red thing here implying that it is not valid

Reply:
Pushed the fix